### DISCUSSION TITLE
Open notifications + control/action centre combined in system tray as Windows 10 flyout

### DISCUSSION THREAD

**Discussion by justaninja** [valinet/ExplorerPatcher#2640] [Ideas]
W11 has the notifications flyout and the action centre separated, whereas these were combined in W10. I think combining these two would be better, because it doesn't make sense to me that the settings icon opens anything other than the settings screen. And the notifications icon opens the calendar, which is pretty pointless since clicking on the time/date already does this. Putting the action centre back in the notifications area and removing the settings icon would make more sense and be more efficient, imo. I don't know why Windows always need to ruin things that worked perfectly fine. Absolutely senseless.

Also, idk if it's me, but there seems to be some extra padding above and underneath the time and date, maybe this could be removed to save space and give a more clean look?

Clicking on the time/date:
![image](https://github.com/valinet/ExplorerPatcher/assets/15679255/bb7659da-4eca-408f-a8f8-5b0b9eb6acdd)

Clicking on the notification icon:
![image](https://github.com/valinet/ExplorerPatcher/assets/15679255/19c86f14-c07e-43cf-85f6-3ae1a84d8a7b)

Clicking on the settings icon:
![image](https://github.com/valinet/ExplorerPatcher/assets/15679255/1e45cb86-6e4e-4304-b8c6-df388a28e504)

Action centre/notifications on W10:
<img width="960" alt="2024-01-02 22_05_24-This PC" src="https://github.com/valinet/ExplorerPatcher/assets/15679255/8932e406-c0d1-431a-a687-46a81b3bd35a">

**Comment by pyrates999** [comment]
That is beyond what EP is for.

**Reply by justaninja** [reply]
Hello, thanks for your answer, but wouldn't this enhance the functionality of the taskbar? Having two icons do the same thing only creates clutter on the taskbar.
I also thought EP was to bring back the look/functionality of the taskbar from older Windows versions? Maybe I'm alone in this, but I think this particular function (action centre and notifications) was much more efficient and better-looking in windows 10 than it is in 11.

**Reply by pyrates999** [reply]
Unless windows 11 has code to combine it, then it can't be added to EP easily.

**Reply by justaninja** [reply]
I see, thanks. That's really a shame.

**Comment by milestprower92** [comment]
> W11 has the notifications flyout and the action centre separated, whereas these were combined in W10. I think combining these two would be better, because it doesn't make sense to me that the settings icon opens anything other than the settings screen. And the notifications icon opens the calendar, which is pretty pointless since clicking on the time/date already does this. Putting the action centre back in the notifications area and removing the settings icon would make more sense and be more efficient, imo. I don't know why Windows always need to ruin things that worked perfectly fine. Absolutely senseless.
> 
> Also, idk if it's me, but there seems to be some extra padding above and underneath the time and date, maybe this could be removed to save space and give a more clean look?
> 
> Clicking on the time/date: ![image](https://private-user-images.githubusercontent.com/15679255/293786568-bb7659da-4eca-408f-a8f8-5b0b9eb6acdd.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MDUwODI5MzIsIm5iZiI6MTcwNTA4MjYzMiwicGF0aCI6Ii8xNTY3OTI1NS8yOTM3ODY1NjgtYmI3NjU5ZGEtNGVjYS00MDhmLWE4ZjgtNWIwYjllYjZhY2RkLnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNDAxMTIlMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjQwMTEyVDE4MDM1MlomWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPTdkZjdhNGMxNDllM2Y0MWUxYTgxZmZkOGY3NzMzMTUxYTdiOTUxMjE0NmM1ZmExNzBhNjM3ODA3MTA4OTU1NGMmWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0JmFjdG9yX2lkPTAma2V5X2lkPTAmcmVwb19pZD0wIn0.UyexF5sL7oH0SFCsljYHAMBdtT8RyXKeB30MmU1sMD0)
> 
> Clicking on the notification icon: ![image](https://private-user-images.githubusercontent.com/15679255/293782494-19c86f14-c07e-43cf-85f6-3ae1a84d8a7b.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MDUwODI5MzIsIm5iZiI6MTcwNTA4MjYzMiwicGF0aCI6Ii8xNTY3OTI1NS8yOTM3ODI0OTQtMTljODZmMTQtYzA3ZS00M2NmLTg1ZjYtM2FlMWE4NGQ4YTdiLnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNDAxMTIlMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjQwMTEyVDE4MDM1MlomWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPTE1YTE5ZWZmZGEyZTkzOWMwOGI4M2M4YmZkZWZmNjhjOTdiYzIzMjFlYmNjODJkMDJhMzcxYWJhZTFhZDQzYTcmWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0JmFjdG9yX2lkPTAma2V5X2lkPTAmcmVwb19pZD0wIn0.2Sizxe3kXDkgxyBl2boTD2qvjS-oqEYvrad0nXEs3vU)
> 
> Clicking on the settings icon: ![image](https://private-user-images.githubusercontent.com/15679255/293789789-1e45cb86-6e4e-4304-b8c6-df388a28e504.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MDUwODI5MzIsIm5iZiI6MTcwNTA4MjYzMiwicGF0aCI6Ii8xNTY3OTI1NS8yOTM3ODk3ODktMWU0NWNiODYtNmU0ZS00MzA0LWI4YzYtZGYzODhhMjhlNTA0LnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNDAxMTIlMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjQwMTEyVDE4MDM1MlomWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPWFiMWQ4ZTMwY2I5ZmEyZDExMjdhZTcwMTczMDBiMzFhMTFlYTgzMzBiNTQ2NjRkNDA5OGUyNWY4ZDk5MDNkMDAmWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0JmFjdG9yX2lkPTAma2V5X2lkPTAmcmVwb19pZD0wIn0.gF1YliA65R8n9X1YIW-tqXsnExxgLWCA71TionXwCDE)
> 
> Action centre/notifications on W10: <img alt="2024-01-02 22_05_24-This PC" width="960" src="https://private-user-images.githubusercontent.com/15679255/293789316-8932e406-c0d1-431a-a687-46a81b3bd35a.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MDUwODI5MzIsIm5iZiI6MTcwNTA4MjYzMiwicGF0aCI6Ii8xNTY3OTI1NS8yOTM3ODkzMTYtODkzMmU0MDYtYzBkMS00MzFhLWE2ODctNDZhODFiM2JkMzVhLnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNDAxMTIlMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjQwMTEyVDE4MDM1MlomWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPTAyNjM1MjA3NjZlMDg2NzM2NDYxZjU2ZmVmMzQ2NGQxMjAzNDBmNjczM2YzNjY1ODA0NWM2Y2RlZDYzZGU5MmQmWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0JmFjdG9yX2lkPTAma2V5X2lkPTAmcmVwb19pZD0wIn0.HFSyfDt9ow1MEo35dOEgbJlS4hyLGOral-RJSFk4Xzk">

if you want the old action center back on windows 11 you have to open regedit, and copy this path:
[HKEY_LOCAL_MACHINE\SOFTWARE\Microsoft\Windows\CurrentVersion]
then right click on current version and create a new key and name it as: Control Center
next create a new dword in the control center folder and name it: UseLiteLayout and the value has to be in 1 

(Issues)
The action center, network, volume, battery, start and search will not combine with the dark mode / light mode.
to fix it you have to set the apps to the dark / light mode option in the settings app

**Reply by pyrates999** [reply]
If this works, then I don't see why it can't be added to EP.  But it's up to the EP developers to decide that.

**Reply by milestprower92** [reply]
because the wifi flyout's buttons breaks and in later builds with the new wifi flyout on the login screen the flyout refuses to work
![Captura de pantalla 2024-01-13 132526](https://github.com/valinet/ExplorerPatcher/assets/131640765/2d1fe2a0-f9b7-4f73-b922-931416bf904d)

**Comment by XuanChen1234** [comment]
@Amrsatrio

**Comment by XuanChen1234** [comment]
@valinet @Vipin2003 Ask for the option to add Windows 10 notifications + control/action center!

**Reply by vippium** [reply]
It is possible but the Old action center is bugged. There is a delay of 4-5 sec while closing. This is hard to resolve. Currently, Search box implementation is going on. Ask Amrsatrio for further updates.
